api/spec/v1.yaml

```diff
@@ -1893,6 +1899,28 @@ components:
       description: |
         An validator registered at the consensus layer.
 
+    Escrow:
+      type: object
+      properties:
+        active_balance:
+          allOf: [$ref: '#/components/schemas/TextBigInt']
+          description: The amount of token actively delegated to this account.
+        active_shares:
+          allOf: [$ref: '#/components/schemas/TextBigInt']
+          description: The shares of tokens delegated to this account.
```

Comment:
a call out to our team, this might need more explanation than "the shares of ..."